source/navigator/RootNavigator.js

```diff
@@ -1,12 +1,14 @@
 import { createSwitchNavigator } from 'react-navigation';
-import { SplashScreen } from 'app/screens';
+import { SplashScreen, EkonomiBistand } from 'app/screens';
+
 import AuthStack from './AuthStack';
 import BottomBarNavigator from './BottomBarNavigator';
 
 export const RootStack = {
   Start: SplashScreen,
   Auth: AuthStack,
   App: BottomBarNavigator,
+  EkonomiScreen: EkonomiBistand,
```

Comment:
This should be named CaseScreen. Since Ekonomisk Bistånd is one of many potential cases :)

Reply:
Suggest a good one :)